### DISCUSSION TITLE
Need explanations to resolve a dependency failure

### DISCUSSION THREAD

**Discussion by antoineDubost** [eclipse-orbit/orbit-simrel#35] [Q&A] [answered]
Hello,

I use https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/release/4.31.0 in a target plateform to retrieve the dependencies of an RCP project.
org.apache.ant in version 1.10.14 is present.
Despite the loading of this dependency, the build fails with this error message :  "The following artifacts could not be resolved: org.eclipse.orbit:ant:jar:1.10.14 (absent) : Could not find artifact org.eclipse.orbit:ant:jar:1.10.14 in Central (https://repo.maven.apache.org/maven2)"

![antIssue](https://github.com/eclipse-orbit/orbit-simrel/assets/163330987/1c0a29a8-22d6-4b10-91e3-ed12cdedc7c9)

Can you give me an explanation and a way to solve this problem? 

Regards,

Antoine

**Comment by merks** [comment, marked as answer]
That sounds like exactly this problem:

https://github.com/eclipse-orbit/orbit-simrel/issues/17

Maybe your maven cache contains the problematic p2 metadata that's been removed since then; the removal of that maven coordinate metadata is exactly why the link to Maven was "missing".    So likely clearing the cache will help.  

There's also a workaround described in the issue because the generated maven artifact is here:

https://repo.eclipse.org/content/repositories/orbit-approved-artifacts/org/eclipse/orbit/ant/1.10.14/

I have no clue why Maven/Tycho insists on trying to find the artifact on maven central when it already has the artifact in hand?!

**Comment by antoineDubost** [comment]
Thank you very much. 
Updating tycho solved my problem.

**Reply by merks** [reply]
Woo hoo.  👍
